From 2ffb87482a77fbbecccc0011b0518837bbaaf964 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Tue, 6 Dec 2011 14:56:00 +0800
Subject: [PATCH 0708/2463] ENGR00162181-1 MX6Q/GPMI : init the GPMI block

The GPMI block SHOULD be initialized when the system starts.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 drivers/mtd/nand/gpmi-nfc/hal-mx50.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mtd/nand/gpmi-nfc/hal-mx50.c b/drivers/mtd/nand/gpmi-nfc/hal-mx50.c
index ee0d318..6545c84 100644
--- a/drivers/mtd/nand/gpmi-nfc/hal-mx50.c
+++ b/drivers/mtd/nand/gpmi-nfc/hal-mx50.c
@@ -339,7 +339,7 @@ static int init(struct gpmi_nfc_data *this)
 	clk_enable(resources->clock);
 
 	/* Reset the GPMI block. */
-	mxs_reset_block(resources->gpmi_regs + HW_GPMI_CTRL0, true);
+	mxs_reset_block(resources->gpmi_regs + HW_GPMI_CTRL0, false);
 
 	/* Choose NAND mode. */
 	__raw_writel(BM_GPMI_CTRL1_GPMI_MODE,
-- 
1.7.7.4

